src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int holder;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        holder = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size > holder) {
+            resize();
+        }
+        Node<K, V> newNode = new Node<>(key, value);
+        int index = getIndex(newNode.key);
+        Node<K, V> nodeFromBucket = table[index];
+        if (nodeFromBucket == null) {
+            table[index] = newNode;
+            size++;
+        } else {
+            while (nodeFromBucket != null) {
+                if (Objects.equals(key, nodeFromBucket.key)) {
+                    nodeFromBucket.value = value;
+                    return;
+                }
+                if (nodeFromBucket.next == null) {
+                    nodeFromBucket.next = newNode;
+                    size++;
+                    return;
```

Comment:
You can move size++ to the and of this method

Reply:
`while (nodeFromBucket != null) {
                if (Objects.equals(key, nodeFromBucket.key)) {
                    nodeFromBucket.value = value;
                    return;
                }`
But we don`t need to do size++ in this block of code